### PR TITLE
first aggregated stats

### PR DESCRIPTION
**Pull request by kremalicious**
- output some basic stats in footer
- fetched on start of app, and refetched every 10 sec. when in viewport 

Screen Shot 2020-11-04 at 10 39 51

numbers fetched from new endpoint 
https://github.com/oceanprotocol/market-stats
https://market-stats.oceanprotocol.com

### CODE REVIEW

File: src/components/molecules/MarketStats.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  owners: number
+  ocean: number
+  datatoken: number
+}
+
+export default function MarketStats(): ReactElement {
+  const [ref, inView] = useInView()
+  const [stats, setStats] = useState<MarketStatsResponse>()
+
+  useEffect(() => {
```
Comment (mihaisc) [inline src/components/molecules/MarketStats.tsx:24]: we should use a canceling token for axios.
```
useEffect(() => {
    const source = axios.CancelToken.source()
    async function getStats() {
      try {
        const response = await axios('https://market-stats.oceanprotocol.com',  { cancelToken: source.token})
        if (!response || response.status !== 200) return
        setStats(response.data)
      } catch (error) {
           if (axios.isCancel(error)) {
              //  cancelled
            } else {
               Logger.error(error.message)
            }
      }
    }

    // Update every 10 sec. when in viewport
    const interval = setInterval(getStats, 10000)

    if (!inView) {
      clearInterval(interval)
    }

    getStats()

    return () => {
          clearInterval(interval)
           source.cancel()
     }
  }, [inView])
```

**Review by mihaisc — APPROVED** [review]

**Review by mihaisc — COMMENTED** [review]
you forgot the error check, without it it will log an error when you cancel the call

**Comment by kremalicious** [comment]
so does it make sense just logging the cancel?

```js
if (axios.isCancel(error)) {
  Logger.log(error.message)
 } else {
  Logger.error(error.message)
}
```

**Comment by mihaisc** [comment]
You can leave it like that, but it doesn't provide much information

**Comment by kremalicious** [comment]
meh, just pushed the change I already committed as that cancel token logging doesn't seem to matter much

**Pull request merged**